app/users/[id]/page.tsx

```diff
@@ -0,0 +1,119 @@
+'use client';
+
+import { ChangeEvent, useState } from 'react';
+
+import { Member } from '~/apis/scheme/member';
+import { Progress } from '~/components/progress';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '~/components/tabs';
+import { Textarea } from '~/components/textarea';
+import useMutationMember from '~/hooks/mutations/useMutationMember';
+import useFetchMember from '~/hooks/queries/useFetchMember';
+import { getAge } from '~/utils/dateFormatter';
+
+import CompleteButton from './_components/complete-button';
+import EditableInput from './_components/editable-input';
+import EditableImage from './_components/editable.image';
+import Info from './_components/info';
+import Review from './_components/review';
+import { TABS_VALUE } from './constants';
+
+const Page = () => {
+  const [isEdit, setIsEdit] = useState(false);
+
+  const { data: member } = useFetchMember();
+  const [inputs, setInputs] = useState({
+    nickname: member?.nickname,
+    introduction: member?.profileImage,
```

Comment:
profileImage대신 introduction이 들어가야 할까요?
```suggestion
    introduction: member?.introduction,
```

Reply:
앗 수정하겠습니다!